src/rtf-plugins/fixture-managers/yarpmanager/YarpFixManager.cpp

```diff
@@ -50,8 +51,14 @@ bool YarpFixManager::setup(int argc, char** argv) {
 
         fixtureName = rf.find("fixture").asString();
         std::string appfile = rf.findFileByName(std::string("fixtures/"+fixtureName).c_str());
-        RTF_ASSERT_ERROR_IF(appfile.size(),
-                            RTF::Asserter::format("yarpmanager cannot find apllication file %s. Is it in the 'fixtures' folder?",
+        if (appfile.empty())
+        {
+            yInfo("yarpmanager: trying to load fixture file from absolute path");
+            appfile = rf.findFileByName(std::string(fixtureName).c_str());
+        }
+
+        RTF_ASSERT_ERROR_IF(!appfile.empty(),
```

Comment:
mmm... due to the rtf issue 67 of robot testing repository ("RTF_TEST_FAIL_IF macro logic is inverted") this line is correct but pretty confusing

Reply:
Go on, just wanted to remark that `size()>0` has an inverted logic wrt `empty()` method 😉